setup.py

```diff
@@ -124,12 +124,12 @@ def finalize_options(self):
                       'doc': ["Sphinx == 1.2.2", "sphinx-bootstrap-theme"]},
       classifiers=classifiers,
       package_data={
+          'skbio.diversity.alpha.tests': ['data/qiime-191-tt/*'],
+          'skbio.diversity.beta.tests': ['data/qiime-191-tt/*'],
           'skbio.io.tests': ['data/*'],
           'skbio.io.format.tests': ['data/*'],
           'skbio.stats.tests': ['data/*'],
           'skbio.stats.distance.tests': ['data/*'],
-          'skbio.diversity.alpha.tests': ['data/*'],
```

Comment:
They are included, I fixed the paths and moved them up to be in alphabetical order.